akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -43,7 +43,15 @@ object ExceptionHandler {
         ctx.complete((status, info.format(settings.verboseErrorMessages)))
       }
       case NonFatal(e) ⇒ ctx ⇒ {
-        ctx.log.error("Error during processing of request: '{}'. Completing with {} response.", e.getMessage, InternalServerError)
+        val message = Option(e.getMessage).getOrElse(s"${e.getClass.getName} (No error message supplied)")
+        val template =
+          s"""Error during processing of request: '{}'.
+             |Completing with {} response.
+             |Stack traces aren't printed by default.
+             |To customize this behavior, consider implementing a custom ExceptionHandler
+             |(http://doc.akka.io/docs/akka-http/current/scala/http/routing-dsl/exception-handling.html)"""
```

Comment:
If we decide that stack trace logging should be added, the last 3 lines become irrelevant and will be removed. Otherwise, I'll just remove the doc link, maybe restating the suggestion about providing a custom handler.